api/s2n.h

```diff
@@ -999,6 +999,12 @@ S2N_API extern int s2n_config_set_verify_host_callback(struct s2n_config *config
  *
  * The default value is 1 if the underlying libCrypto implementation supports OCSP. 
  *
+ * @note SHA-1 is the only supported hash algorithm for the `certID` field. If a different hash 
+ * algorithm is used, validation will fail and manual verification will be needed. This is 
```

Comment:
Do you think it's worth mentioning disabling OCSP verification as a workaround? It seems so unlikely that users would need to do this, that I'm wondering if it's not worth potentially confusing them by suggesting they opt-out of the s2n-tls verification and implement their own. Not sure, though. Minimally, I think we'd probably want to mention that this would likely never be needed. 

Also, it's currently not super clear what the user would actually do here. Like, if validation fails, `s2n_negotiate()` will error and manual verification won't be possible. Maybe we should explicitly mention why `s2n_config_set_check_stapled_ocsp_response()` is needed?

Reply:
Hmm I agree that disabling OCSP verification is an unlikely scenario for most users, but I believe it's still valuable to document an alternative they can apply. I have updated the comment to indicate that this is a very unlikely scenario and clarified the steps needed if validation fails.